code/game/objects/items/implants/implant_stealth.dm

```diff
@@ -37,6 +37,9 @@
 
 /obj/structure/closet/cardboard/agent/Bump(atom/A)
 	. = ..()
+	if(istype(A, /obj/machinery/door))
+		for(var/mob/mob_in_box as anything in contents)
+			A.Bumped(mob_in_box)
```

Comment:
```suggestion
	if(istype(A, /obj/machinery/door))
		for(var/mob/mob_in_box in contents)
			A.Bumped(mob_in_box)
```

`contents` is not guaranteed to be a list of only mobs so let's avoid `as anything` and err on the side of caution with the typecheck. probably really wasteful to have to call Bumped() on useless stuff like that too